Vagrantfile

```diff
@@ -32,6 +32,20 @@ unless File.exist?("#{dir}/config.yml")
 end
 vconfig = YAML.load_file("#{dir}/config.yml")
 
+# Unless it's specifically set, eg. for a vagrant deployment plugin, use local as
+# the environment.
+env = vconfig.include?('env') ? vconfig['env'] : 'local'
+vconfig['extra_configs'] ||= []
+# Try including an environment specific config file.
+vconfig['extra_configs'].push("#{env}.config.yml")
+# Load all custom config files.
+vconfig['extra_configs'].each do |config_file|
+  if File.exist?(config_file)
```

Comment:
todo: this check should not be present for the manually defined `extra_configs` as ansible would fail anyways.